### PR TITLE
Codecov in README

### PR DESCRIPTION
**Pull request by vishal-biyani**
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1309)

### CODE REVIEW

**Comment by life1347** [comment]
I'm thinking we better to add the status icon after we add more unit tests to the codebase. What do you think?

**Comment by vishal-biyani** [comment]
I am actually ok adding now too - it will act as a push for us to increase coverage. WDYT?

**Comment by life1347** [comment]
> it will act as a push for us to increase coverage

ok, that's fair.

File: README.md [hunk 1]
```diff
@@ -2,8 +2,9 @@ Fission: Serverless Functions for Kubernetes
 ============================================
 [![Build Status](https://travis-ci.org/fission/fission.svg?branch=master)](https://travis-ci.org/fission/fission)
 [![Go Report Card](https://goreportcard.com/badge/github.com/fission/fission)](https://goreportcard.com/report/github.com/fission/fission)
-[![Fission Slack](http://slack.fission.io/badge.svg)](http://slack.fission.io)
+[![codecov](https://codecov.io/gh/fission/fission/branch/master/graph/badge.svg)](https://codecov.io/gh/fission/fission)
```
Comment (life1347) [inline README.md]: Please move the blank line between slack and fission.io link, so that we can keep icons in one line.

**Comment by vishal-biyani** [comment]
Closing in favor of https://github.com/fission/fission/pull/1604

**Pull request closed without merging**